zerver/views/streams.py

```diff
@@ -671,7 +672,9 @@ def add_subscriptions_backend(
         authorized_streams,
         unauthorized_streams,
         streams_to_which_user_cannot_add_subscribers,
-    ) = filter_stream_authorization(user_profile, existing_streams, is_subscribing_other_users)
+    ) = asdict(
+        filter_stream_authorization(user_profile, existing_streams, is_subscribing_other_users)
+    ).values()
```

Comment:
This feels fragile; would be better to just use the dataclass in this function.

Reply:
Fixed!